app/views/users/_form.html.erb

```diff
@@ -69,12 +69,28 @@
     <div class='tab-pane' id='roles'>
       <%= checkbox_f f, :admin if User.current.can_change_admin_flag? %>
       <%= multiple_checkboxes f, :roles, @user, Role.givable.for_current_user,
-                               { :label => _('Roles')}, {:disabled => @editing_self ? Role.givable.for_current_user.pluck(:id) : false } %>
+                              { :label => _('Roles')}, {:disabled => @editing_self ? Role.givable.for_current_user.pluck(:id) : false } %>
+      <div class="form-group">
+        <% if @user.cached_usergroups.uniq.any? %>
```

Comment:
Please make this `if` condition show/hide the entire div, not just the title. Also use `.usergroups`, not `.cached_usergroups` - and no need for `.uniq` as far as I know